drip/models.py

```diff
@@ -5,6 +5,13 @@
 from django.conf import settings
 
 from drip.utils import get_user_model
+from .types import (
+    AbstractQuerySetRuleQuerySet,
```

Comment:
Is this name ok? Why does have `QuerySet` twice?

Reply:
Because this defines a QuerySet of `AbstractQuerySetRule`... it might sound confusing but the pattern I tried to propose is `ClassNameQuerySet`